### DISCUSSION TITLE
Constantly getting an error when I try to clone repos

### DISCUSSION THREAD

**Discussion by NehaMahendraSingh** [community/community#40586] [Repositories]
### Select Topic Area

Product Feedback

### Body

remote: Write access to repository not granted.
fatal: unable to access 'https://github.com/****/': The requested URL returned error: 403

**Comment by DSDmark** [comment]
Hi @NehaMahendraSingh,

about your error. In my opinion, there is the several cases like:- 

1. maybe your **URL** or **origin** which you try to clone is wrong.

2. for all authenticated Git operations. Beginning August 13, 2021, we will no longer accept account passwords when authenticating Git operations on GitHub.com. [Token authentication to access GitHub](https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/) if you are using HTTPS. So you must [generate an access token](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) in your profile settings page and use it as your password. Check the boxes necessary to write to the repository.

**Comment by ghostinhershell** [comment]
Hi @NehaMahendraSingh,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your duplicate post in the Accessibility category [here](https://github.com/orgs/community/discussions/40587) was removed because it contained content that violates our Community Code of Conduct, specifically posting the same question in multiple places. As a result, we will delete your duplicate post and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
